GOVERNANCE.md

```diff
@@ -0,0 +1,84 @@
+
+External collaborators may be given write-access to specific repositories on a case-by-case basis at the discretion of the repository maintainer. Their status in the GitHub organization is ["outside collaborator"](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization).
+
+### Team organizational chart
+
+![Organizational chart of the Epiverse-TRACE project.](org_chart.png)
+
+### Former team members
+
+When Epiverse-TRACE team members change employment, they are removed from the organization and re-invited as an outside collaborator on the repositories they are currently contributing to. It is the responsibility of each respective partner to reflect internal changes in the organization as soon as possible.
+
+## Definition of Coding & Packaging Standards
+
+### Scope and Location
+
+The coding and packaging standards apply to all R packages hosted on this GitHub organization. The standards are defined in the [`blueprints`](https://github.com/epiverse-trace/blueprints) repository, in the form of an online book.
+The Blueprints aim to ensure consistency and quality control in the artifacts produced by the project (e.g., R packages), rather than homogeneity in the working patterns and processes of individual teams.
+
+Whenever possible, the blueprints standard are implemented in the [`packagetemplate` template GitHub repository](https://github.com/epiverse-trace/packagetemplate) to facilitate adoption throughout the project.
+
+### Decision-making process
+
+The blueprints are continuously reviewed and improved by the blueprints working group, on the ["rough consensus" model](https://en.wikipedia.org/wiki/Rough_consensus), pioneered by the IETF.
+All new proposals for the blueprints are initially discussed in blueprints working group private slack channel for simple changes, or via synchronous meetings for contentious or more difficult changes. Results of these discussions are then encoded in a pull request in the blueprints repository. The link to the pull request is shared on the private slack channel for review, and all working group members have 2 weeks to voice concerns or request changes before the pull request is merged.
+
+The blueprints working group is currently composed of a senior representative of each team involved in the project, as well as one external advisor:
+
+- Sebastian Funk (representing the LSHTM-MRCG team)
+- Chris Hartgerink (representing data.org)
+- Hugo Gruson (representing data.org)
+- Jaime Pavlich (representing the TRACE-LAC team)
+- Tim Taylor (external advisor)
```

Comment:
```suggestion
- Chris Hartgerink (representing data.org)
- Hugo Gruson (representing data.org)
- Jaime Pavlich (representing the TRACE-LAC team)
- Sebastian Funk (representing the LSHTM-MRCG team)
- Tim Taylor (external advisor)
```

Again the alternative for alphabetical sort in case preferred.